### PR TITLE
DOCSP-43306-ToC-Refactor

### PR DESCRIPTION
**Pull request by ltran-mdb2**
## DESCRIPTION

Updated ToC based on the [new ToC style guidelines](https://www.mongodb.com/docs/meta/style-guide/style/toc-labels/)

## STAGING

https://deploy-preview-164--docs-relational-migrator.netlify.app
all ToC labels can be viewed through the menu drop downs

## JIRA

https://jira.mongodb.org/browse/DOCSP-43306

## BUILD LOG

https://app.netlify.com/sites/docs-relational-migrator/deploys/66e0af9c2d167100071f5c97

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/installation.txt [hunk 1]
```diff
@@ -68,10 +68,10 @@ For instructions on installing Relational Migrator, see the following pages:
...
-   /installation/deployment-considerations
-   /installation/install-on-a-local-machine/install-on-a-local-machine
-   /installation/install-on-an-unattended-server/install-on-an-unattended-server
-   /installation/kafka-deployments/migrator-with-kafka
-   /installation/install-with-docker
-   /installation/file-location
+   System Requirements </installation/system-requirements>
+   Deployment Considerations </installation/deployment-considerations>
+   Install Locally </installation/install-on-a-local-machine/install-on-a-local-machine>
+   Install on an Unattended Server </installation/install-on-an-unattended-server/install-on-an-unattended-server>
```
Comment (kennethdyer) [inline source/installation.txt:74]: [Suggestion]: Install on Unattended Server

File: source/installation/install-on-a-local-machine/install-on-a-local-machine.txt [hunk 2]
```diff
@@ -71,7 +71,7 @@ Learn More
    :hidden:
    :titlesonly:
 
-   /installation/install-on-a-local-machine/install-windows
-   /installation/install-on-a-local-machine/install-mac
-   /installation/install-on-a-local-machine/install-ubuntu
-   /installation/install-on-a-local-machine/install-rhel
+   Windows </installation/install-on-a-local-machine/install-windows>
+   Mac </installation/install-on-a-local-machine/install-mac>
```
Comment (kennethdyer) [inline source/installation/install-on-a-local-machine/install-on-a-local-machine.txt:75]: [Suggestion]: As of a few releases ago, Apple now styles their operating system as `macOS`, (q.e.d., [Wiki](https://en.wikipedia.org/wiki/MacOS)).

File: source/installation/install-on-an-unattended-server/debian-server-installation/debian-server-installation.txt [hunk 3]
```diff
@@ -27,5 +27,5 @@ Get Started
    :hidden:
    :titlesonly:
 
-   /installation/install-on-an-unattended-server/debian-server-installation/install-debian-server
-   /installation/install-on-an-unattended-server/debian-server-installation/debian-system-service
+   Install </installation/install-on-an-unattended-server/debian-server-installation/install-debian-server>
+   Run as a System Service </installation/install-on-an-unattended-server/debian-server-installation/debian-system-service>
```
Comment (kennethdyer) [inline source/installation/install-on-an-unattended-server/debian-server-installation/debian-server-installation.txt:31]: [Suggestion] Drop the indefinite article for succinctness: "Run as System Service".

File: source/installation/install-on-an-unattended-server/rhel-centos-server-installation/rhel-centos-server-installation.txt [hunk 4]
```diff
@@ -26,5 +26,5 @@ Get Started
    :titlesonly:
    :hidden:
 
-   /installation/install-on-an-unattended-server/rhel-centos-server-installation/install-rhel-server
-   /installation/install-on-an-unattended-server/rhel-centos-server-installation/rhel-system-service
+   Install </installation/install-on-an-unattended-server/rhel-centos-server-installation/install-rhel-server>
+   Run as a System Service </installation/install-on-an-unattended-server/rhel-centos-server-installation/rhel-system-service>
```
Comment (kennethdyer) [inline source/installation/install-on-an-unattended-server/rhel-centos-server-installation/rhel-centos-server-installation.txt:30]: [Suggestion]: Run as System Service

File: source/installation/install-on-an-unattended-server/windows-server-installation/windows-server-installation.txt [hunk 5]
```diff
@@ -28,6 +28,6 @@ Get Started
    :titlesonly:
    :hidden:
 
-   /installation/install-on-an-unattended-server/windows-server-installation/install-windows-server
-   /installation/install-on-an-unattended-server/windows-server-installation/windows-system-service
-   /installation/install-on-an-unattended-server/windows-server-installation/windows-external-access
+   Install </installation/install-on-an-unattended-server/windows-server-installation/install-windows-server>
+   Run as a System Service </installation/install-on-an-unattended-server/windows-server-installation/windows-system-service>
```
Comment (kennethdyer) [inline source/installation/install-on-an-unattended-server/windows-server-installation/windows-server-installation.txt:32]: Ditto

File: source/installation/kafka-deployments/migrator-with-kafka.txt [hunk 6]
```diff
@@ -86,7 +86,7 @@ Learn More
    :hidden:
    :titlesonly:
 
-   /installation/kafka-deployments/install-kafka-cluster
-   /installation/kafka-deployments/install-docker
-   /installation/kafka-deployments/install-confluent-kafka
-   /installation/kafka-deployments/advanced-settings
+   Use Existing Kafka Cluster </installation/kafka-deployments/install-kafka-cluster>
+   Use a Containerized Cluster with Docker </installation/kafka-deployments/install-docker>
```
Comment (kennethdyer) [inline source/installation/kafka-deployments/migrator-with-kafka.txt:90]: [Suggestion]: Drop the indefinite article.

File: source/mapping-rules/mapping-rule-options/mapping-rule-options.txt [hunk 7]
```diff
@@ -95,9 +95,10 @@ mapping rule option.
...
-   /mapping-rules/mapping-rule-options/embedded-array
-   /mapping-rules/mapping-rule-options/embedded-documents
-   /mapping-rules/mapping-rule-options/mapping-rule-filters
-   /mapping-rules/mapping-rule-options/time-series
-   /mapping-rules/mapping-rule-options/rename-collection
+   New Documents </mapping-rules/mapping-rule-options/new-documents>
+   Embedded Array </mapping-rules/mapping-rule-options/embedded-array>
+   Embedded Documents </mapping-rules/mapping-rule-options/embedded-documents>
+   Mapping Rule Filters </mapping-rules/mapping-rule-options/mapping-rule-filters>
+   Rename a Collection </mapping-rules/mapping-rule-options/rename-collection>
```
Comment (kennethdyer) [inline source/mapping-rules/mapping-rule-options/mapping-rule-options.txt:102]: [Suggestion]: Rename Collection

Comment (kennethdyer) [reply source/mapping-rules/mapping-rule-options/mapping-rule-options.txt:102]: And for next line: Configure Time Series Collection.

File: source/projects/projects.txt [hunk 8]
```diff
@@ -40,13 +40,13 @@ There are multiple options for you to create a project:
...
-   /projects/configure-settings
-   /projects/manage-relational-connection
-   /projects/import-project
-   /projects/export-project
-   /projects/deleting-projects
-   /projects/renaming-projects
+   Settings </projects/project-settings>
+   Create with a Live Database </projects/create-project-live-connection>
+   Create by Loading a Schema </projects/create-project-loading-schema-files>
+   Create with a Sample Schema </projects/create-project-sample-schema>
```
Comment (kennethdyer) [inline source/projects/projects.txt:46]: [Suggestion]: Trim indefinite articles on three lines above.

**Review by kennethdyer — APPROVED** [review]
Thanks for these changes @ltran-mdb2.  This LGTM.  I left a few comments on indefinite articles, which I think you can trim without losing in titular semantics.

File: source/installation.txt (hunk 1 above)
Comment (ltran-mdb2) [reply source/installation.txt]: Done

File: source/installation/install-on-a-local-machine/install-on-a-local-machine.txt (hunk 2 above)
Comment (ltran-mdb2) [reply source/installation/install-on-a-local-machine/install-on-a-local-machine.txt]: Done

File: source/installation/install-on-an-unattended-server/debian-server-installation/debian-server-installation.txt (hunk 3 above)
Comment (ltran-mdb2) [reply source/installation/install-on-an-unattended-server/debian-server-installation/debian-server-installation.txt]: Done

File: source/installation/install-on-an-unattended-server/rhel-centos-server-installation/rhel-centos-server-installation.txt (hunk 4 above)
Comment (ltran-mdb2) [reply source/installation/install-on-an-unattended-server/rhel-centos-server-installation/rhel-centos-server-installation.txt]: Done

File: source/installation/kafka-deployments/migrator-with-kafka.txt (hunk 6 above)
Comment (ltran-mdb2) [reply source/installation/kafka-deployments/migrator-with-kafka.txt]: Done

File: source/mapping-rules/mapping-rule-options/mapping-rule-options.txt (hunk 7 above)
Comment (ltran-mdb2) [reply source/mapping-rules/mapping-rule-options/mapping-rule-options.txt]: Done

File: source/projects/projects.txt (hunk 8 above)
Comment (ltran-mdb2) [reply source/projects/projects.txt]: Done

File: source/installation/install-on-an-unattended-server/windows-server-installation/windows-server-installation.txt (hunk 5 above)
Comment (ltran-mdb2) [reply source/installation/install-on-an-unattended-server/windows-server-installation/windows-server-installation.txt]: Done

**Comment by ltran-mdb2** [comment]
Hi @sarahemlin ! Could you give this Relational Migrator ToC Refactor an external review?

**Review by sarahemlin — APPROVED** [review]

**Pull request merged**